config/routes.rb

```diff
@@ -2,6 +2,11 @@
 
   root 'welcome#index'
 
+  namespace :api, path: nil, constraints: {subdomain: 'api'}, defaults: {format: :json} do
```

Comment:
Line is too long. [90/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.